_data/staff.yml

```diff
@@ -10,12 +10,18 @@ administrators:
     - username: ElectroidFilms
       id: dad8b95c-cf6a-44df-982e-8c8dd70201e0
       role: SysAdmin & Development
+      links:
+        github: Electroid
     - username: applenick
       id: f54a1b17-3167-4162-948a-9c95d65a18a5
       role: SysAdmin & Development
+      links:
+        github: applenick
```

Comment:
Interesting choice! I personally don't mind my GitHub being linked, but hopefully that's the same for everyone else.

Reply:
I hope so as well. I was able to find people's GitHub either from looking at their Discord profile or looking at prior commits in other OCC or PGM repositories. I figured it would make sense that people that are actively involved with OCC development should have a link to direct visitors to their official GH profile.